app/komponenter/hovedInnhold/HovedInnhold.tsx

```diff
@@ -1,14 +1,17 @@
+import { useTekster } from '~/hooks/contextHooks';
 import Banner from '../banner/Banner';
 import css from './hovedInnhold.module.css';
+import { ESanityMappe } from '~/typer/felles';
 
 interface Props {
   children?: React.ReactNode;
 }
 
 const HovedInnhold: React.FC<Props> = ({ children }) => {
+  const { bannerTekst } = useTekster(ESanityMappe.FELLES);
   return (
     <>
-      <Banner />
+      <Banner bannerTekst={bannerTekst} />
```

Comment:
```suggestion
      <Banner tekst={bannerTekst} />
```

Siden denne propen ligger på komponenten `Banner` trenger den nok bare å hete `tekst`. Eventuelt er det vanlig å bruke `children` propen også når man skal ha inn tekst

Reply:
Eksempel med children:

`<Banner> bannertekst som vises i banneret </ Banner>`
Da vil teksten bli tilgjengelig inne i `Banner` gjennom `children` komponenten

Reply:
Eniig. Endret her: [b30191c](https://github.com/bekk/nav-familie-endringsmelding/pull/45/commits/b30191ca0012ba1ce0265a461d4a78bc44ca6619)